core/textile/sync/synchronizer.go

```diff
@@ -243,6 +257,8 @@ func (s *synchronizer) executeTask(ctx context.Context, t *Task) error {
 		err = s.processBucketBackupOn(ctx, t)
 	case bucketBackupOffTask:
 		err = s.processBucketBackupOff(ctx, t)
+	case bucketRestoreTask:
```

Comment:
Do we need one for the file task as well?

Reply:
@jsonsivar 

Yes, indeed. Thank you for spotting this. Fixed in https://github.com/FleekHQ/space-daemon/pull/207/commits/2bcda3d1052a5da450071ec4301631a821e8b0ff#diff-47b13a650b4b80008ff5c0fde8630023ebe86568cc4ca509a299cd30b29cfa61R262